powermeter/slewrate.py

```
@@ -0,0 +1,71 @@
+    
+            if self.last_values is None or self.last_time is None:
+                # First reading – initialise without slewing
+                self.last_values = list(raw_values)
+                self.last_time = current_time
+                return list(raw_values)
+    
+            elapsed = current_time - self.last_time
+            max_change = self.slew_rate_watts_per_sec * elapsed
+    
+            slewed = []
+            for raw, last in zip(raw_values, self.last_values):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)